dali/python/nvidia/dali/_utils/autoserialize.py

```diff
@@ -30,7 +30,12 @@ def _discover_autoserialize(module, visited):
     """
     assert module is not None
     ret = []
-    module_members = inspect.getmembers(module)
+    try:
```

Comment:
Can you add a simple test with `import xyz` inside?

Reply:
Done